test/options/lines-between-rulesets-css/test.js

```diff
@@ -0,0 +1,24 @@
+/* global describe, it */
+'use strict';
+
+describe('options/lines-between-rulesets:css', function() {
```

Comment:
@AoDev Thanks for your prompt reply. I'll do my best to address the review comments this week.